File: system-test/sec-tests/java/Sec45PathTraversal.java
// SEC-45: Path Traversal
// Test: TC-CR-049
package com.example;

import java.nio.file.*;

public class FileDownloadService {

    public Path getFilePath(String fileName) {
        return Paths.get("/var/data/", fileName);  // User input not sanitized
    }
}

File: system-test/sec-tests/javascript/sec_45_path_traversal.js
// SEC-45: Path Traversal
// Test: TC-CR-049
const express = require('express');
const path = require('path');
const fs = require('fs');
const app = express();

app.get('/download', (req, res) => {
  const fileName = req.query.file;
  const filePath = path.join('/var/data/', fileName);  // User input not sanitized
  res.download(filePath);
});
